README.md

```diff
@@ -200,26 +205,27 @@ This will generate:
 - `dist/bindings/android/e4.aar`: the Android package, containing compiled Java class and native libraries for most common architectures
 - `dist/bindings/android/e4-sources.jar`: the Java source files
 
-After importing the AAR in your project, E4 client can be created and invoked 
+After importing the AAR in your project, E4 client can be created and invoked
 in a similar way than the Go version, for example using Kotlin:
 
 ```kotlin
-import io.teserakt.e4.*
-import io.teserakt.crypto.*
+import io.teserakt.e4.E4
+import io.teserakt.e4.SymNameAndPassword
+import io.teserakt.crypto.Crypto
 
 val cfg = SymNameAndPassword()
 cfg.name = "deviceXYZ"
 cfg.password = "secretForDeviceXYZ"
 
-val client = E4.newClient(cfg,  cacheDir.path + "deviceXYZ.json")
+val state = ByteArray(0)
+val client = E4.newClient(cfg, E4.newMemoryStore(state))
```

Comment:
We probably want a file store as well for demo purposes, right? Otherwise we have no persistence and that might be confusing for users (hence why a silly toy filestore exists in libe4 - to show people it's possible). Also we probably want it for ourselves (test clients).

I'm updating this comment to say: perhaps the readme should show for example the use of files, which is the more common use case, but also explain how you can use another storage type?